.gitignore

```diff
@@ -5,4 +5,6 @@ dist
 test-lib
 coverage
 .nyc_output
-.idea
\ No newline at end of file
+.idea
+yarn.lock
```

Comment:
yarn.lock should *not* be in git ignore. Assuming we pick yarn for now as primarly way to maintain the repo